docs/existing_data.rst

```diff
@@ -34,3 +34,7 @@ FAQ
 - After asking Patroni to restart the node, PostgreSQL displays the error message ``could not open configuration file "/etc/postgresql/10/main/pg_hba.conf": No such file or directory``
 
   It can mean various things depending on how you manage PostgreSQL configuration. If you specified `postgresql.config_dir`, Patroni generates the ``pg_hba.conf`` based on the settings in the :ref:`bootstrap <bootstrap_settings>` section only when it bootstraps a new cluster. In this scenario the ``PGDATA`` was not empty, therefore no bootstrap happened. This file must exist beforehand.
+
+- What is the minimal number of PostgreSQL nodes in a Patroni cluster?
+
+  Patroni/PostgreSQL nodes are decoupled from DCS nodes (except when Patroni implements RAFT on its own) and therefore there is no requirement on minimal number of nodes.
```

Comment:
The `existing_data.rst` is not the best place for it. Probably it should go somewhere to https://github.com/zalando/patroni/blob/master/docs/README.rst#running-and-configuring

Reply:
Please disregard this pull request, I've created a new one for README.rst